diceware/generate_test.go

```diff
@@ -1,7 +1,12 @@
 package diceware
 
 import (
+	"crypto/sha256"
+	"fmt"
+	"reflect"
 	"testing"
+
+	"golang.org/x/crypto/hkdf"
```

Comment:
Is there a rand source in the stdlib we can use? This only seems to be for tests, but I really want to rely on no external deps.

Reply:
Pulling in from x/crypto is probably "safe", but I like having zero deps